src/scripts/interface/elementIds.ts

```
@@ -0,0 +1,11 @@
+
+export class ElementIds {
+  public static readonly dropdownDialog = 'dropdown-dialog';
+  public static readonly restartDialog = 'restart-dialog';
+
+  public static readonly dropdownButton = 'dropdown-button';
+  public static readonly undoButton = 'undo-button';
+  public static readonly cycleButton = 'cycle-button';
+
+  public static readonly gameContainer = 'game-container';
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)